LOCAL_RUN.md

```diff
@@ -0,0 +1,153 @@
+
+If you're interested to know how to run Juju on your cloud of choice, checkout [the official docs](https://juju.is/docs/olm/clouds); you can always run `juju clouds` to check your configured clouds. In the instructions below, we will always use `microk8s`, but you can replace it with the name of the cloud you're using.
+
+## Bootstrap Juju
+In Juju terms, "bootstrap" means "create a Juju controller", which is the part of Juju that runs in your cluster and controls the applications.
+
+```bash
+juju bootstrap microk8s finos-legend-controller
+```
+
+You are free to use any name other than `finos-legend-controller` but it must be consistent through the rest of this tutorial.
+
+## Add the Juju Legend Model
+[Juju models](https://juju.is/docs/olm/models) are a logical grouping of applications and infrastructure that work together to deliver a service or product. In Kubernetes terms, models are effectively namespaces. Models are fundamental concepts in Juju and implement service isolation, access control, repeatability and boundaries.
+
+You can add a new model with
+``` bash
+juju add-model finos-legend
+```
+
+## Deploy the Legend Bundle
+When you deploy an application with Juju, the installation code in the charmed operator will run and set up all the resources and environmental variables needed for the application to run properly. In the case of this tutorial, we are deploying a *bundle*, which describes applications to be deployed and relationships between them.
+
+Deploy the [finos-legend-bundle](https://github.com/finos/finos-legend-bundle) using `juju deploy finos-legend-bundle`.
+
+In another terminal, you can check the deployment status and the integration code running using `watch --color juju status --color`.
+
+You'll notice that the Unit `finos-legend-gitlab-integrator-k8s/0` will get to `blocked` status; this is expected, as you'll need to [Setup and Configure GitLab](#Setup-and-Configure-GitLab).
+
+## Setup and Configure GitLab
+To run Legend, you need to either run a GitLab instance somewhere, or use GitLab.com; the type of installation really depends on user's requirements, there is a secion in `LOCAL_GITLAB.md` that talks about that (TODO).
+
+If this is your first experience with Legend, we suggest you starting with GitLab.com; you simply need to create an account and you're good to go. If, instead, you're interested to test a Legend deployment with a local GitLab, please follow instructions on `LOCAL_GITLAB.md`.
+
+To configure GitLab with Legend, you'll need a [GitLab Access Token](LOCAL_RUN.md):
+```bash
+$ juju config finos-legend-gitlab-integrator-k8s access-token="<GITLAB_ACCESS_TOKEN>"
+```
+
+`GITLAB_ACCESS_TOKEN` is what we've generated earlier; you can add `gitlab-host=<GITLAB_HOST>` to the command, if you want to point to the host/DNS/IP of another GitLab instance.
```

Comment:
I worry "another GitLab instance" may mislead here. Should we emphasise that the access token must be valid for the specific Gitlab instance pointed to by `gitlab-host=`.

Reply:
Text updated